### DISCUSSION TITLE
Exporting DORA reports and dashboards

### DISCUSSION THREAD

**Discussion by debashish1976** [apache/devlake#6785] [Ideas]
Hello,
How do we consume devlake reports, recommendations or dashboards in other software? This is a great tool. But I would like the reports to be integrated with some of our other reports and dashboards that we are managing. Any API integration would be ideal. 

Note: I am not looking at data source integration to push metrics to devlake. I looking to push data out of devlake.

**Comment by klesh** [comment]
Hi, you may connect to the database and export whatever you need, all data collected/calculated are stored in it.
